docs/azure/ai/get-started-azure-openai.md

```diff
@@ -0,0 +1,98 @@
+1. From a terminal or command prompt, navigate to the `01-HikerAI` directory.
+1. Create the Azure resources using the Azure Developer CLI:
+	```bash
+	azd up
+	```
+1. It's now time to try the console application. Depending on your Azure subscription it's possible that a few (~5) minutes more minute are required before the model deployed in Azure OpenAI get available. If you get an error message about this, wait a few minutes and try again.
+	```bash
+	dotnet run
+	```
+
+## How it works
+
+The client library is available through NuGet, as the `Azure.AI.OpenAI` package. Using the client we can send and receive requests to an Azure OpenAI service deployed in Azure.
+
+Once the `OpenAIClient` client is created, we provide more context to the model by adding a system prompt. This instructs the model how you'd like it to act during the conversation.
+
+```csharp
+var systemPrompt = 
+"""
+You are a hiking enthusiast who helps people discover fun hikes in their area. You are upbeat and friendly. 
+You introduce yourself when first saying hello. When helping people out, you always ask them 
+for this information to inform the hiking recommendation you provide:
+
+1. Where they are located
+2. What hiking intensity they are looking for
+
+You will then provide three suggestions for nearby hikes that vary in length after you get that information. 
+You will also share an interesting fact about the local nature on the hikes when making a recommendation.
+""";
+
+completionOptions.Messages.Add(new ChatRequestSystemMessage(systemPrompt));
+```
+
+Then we can add a user message to the model, and get the response from the model.
+
+```csharp
+string userGreeting = """
+Hi! 
+Apparently you can help me find a hike that I will like?
+""";
+
+completionOptions.Messages.Add(new ChatRequestUserMessage(userGreeting));
+Console.WriteLine($"\n\nUser >>> {userGreeting}");
+
+ChatCompletions response = await openAIClient.GetChatCompletionsAsync(completionOptions);
+ChatResponseMessage assistantResponse = response.Choices[0].Message;
+Console.WriteLine($"\n\nAI >>> {assistantResponse.Content}");
+completionOptions.Messages.Add(new ChatRequestSystemMessage(assistantResponse.Content)); 
+```
+
+Try it out customize the prompt and message and see how the AI model can help you find a hike that you will like.
+
+
+## Clean up resources
+
+When you no longer need the sample application or resources, remove the corresponding deployment and all resources.
+```bash
+azd down
+```
+
+## Troubleshooting
+
+On Windows, you may get an error message: "*postprovision.ps1 is not digitally signed. The script will not execute on the system*" after the deployment. This is cause by the script "postprovision" being executed locally after the deployment to create .NET secret that will be used in the application. To avoid this error, execute the command `Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass`. And re-run the `azd up` command.
```

Comment:
```suggestion
On Windows, after running `azd up` you may get the following error message:
> *postprovision.ps1 is not digitally signed. The script will not execute on the system* 

The script **postprovision.ps1** is executed to set the .NET user secrets used in the application. To avoid this error, run the following Powershell command:

    ```powershell
    Set-ExecutionPolicy -Scope Process -ExecutionPolicy Bypass
    ```

Then re-run the `azd up` command.
```